src/Client.php

```diff
@@ -129,26 +129,26 @@ public function getAccountDetails()
 
     public function isSubscribed($listId, $email)
     {
-        $email = strtolower($email);
-        $hash = md5($email);
-        $endpoint = sprintf('lists/%s/members/%s', $listId, $hash);
+        return 'subscribed' === $this->getSubscriberStatus($listId, $email);
+    }
+
+    public function getSubscriberStatus($listId, $email)
+    {
+        $endpoint = sprintf('lists/%s/members/%s', $listId, $this->subscriberHash($email));
 
         $response = $this->get($endpoint);
 
         $body = json_decode($response->getBody());
 
-        if ('subscribed' === $body->status) {
-            return true;
-        }
-
-        return false;
+        return $body->status;
     }
 
     public function subscribeToList($listId, $email, $mergeVars = [], $doubleOptin = true, $interests = [])
     {
+        $status = $this->getSubscriberStatus($listId, $email);
         $endpoint = sprintf('lists/%s/members', $listId);
 
-        if (!$this->isSubscribed($listId, $email)) {
+        if ('subscribed' !== $status) {
```

Comment:
Would it make sense to use a class constant for subscribed?

Reply:
Yeah, something like `const STATUS_SUBSCRIBED = 'subscribed';`?

Reply:
Yep!

Reply:
done